### PR TITLE
add lipidorderkit to mdanalysis

### PR DESCRIPTION
**Pull request by ricard1997**
This code allows for the computation of lipid order parameters for all-atom MDAsimulation for researchers that study membrane simulations.

### CODE REVIEW

File: mdakits/lipidorderkit/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+##   mamba -c conda-forge install MYPROJECT
+## for a conda package installation).
+## Here we use a simple PyPi installation:
+install:
+  - pip install -e .
+
+## List(str): a list of commands to use when installing the mdakit from its
+## source code.
+src_install:
+  - pip install git+https://github.com/ricard1997/lipidorderkit@main
```
Comment (lilyminium) [inline mdakits/lipidorderkit/metadata.yaml:72]:
```suggestion
  - git clone https://github.com/ricard1997/lipidorderkit.git
  - cd lipidorderkit && pip install . && cd ..
```

These commands clone the repo and then install the package from source.

**Review by lilyminium — CHANGES_REQUESTED** [review]
Hi @ricard1997, thanks for adding your MDAKit to the registry -- exciting to see more lipid analyses here! I've just done a quick first pass over the metadata file and commented on where some entries need some modification to be properly parsed by the MDAKits registry.

File: mdakits/lipidorderkit/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+## str: the link to the project's code
+## Please note that this is not limited to GitHub! Can be Gitlab, etc..
+project_home: https://github.com/ricard1997/lipidorderkit/
+
+## str: the link to the project's documentation
+documentation_home: https://lipidorderdocs.readthedocs.io/en/latest/
+
+## str: the type of documentation available [UserGuide, API, README]
+documentation_type: UserGuide + API
```
[the remark this answers is not in the document]
Comment (ricard1997) [reply mdakits/lipidorderkit/metadata.yaml:55]: Yes, once the MDAKit is accepted I plan to do a full tutorial and an explanation about lipid order parameters.

File: mdakits/lipidorderkit/metadata.yaml [hunk 3]
```diff
@@ -0,0 +1,122 @@
+# TEMPLATE MDAKit file
+# --------------------
+# 
+# Please replace ALL entries with appropriate content for YOUR MDAKit.
+# Below we use the placeholder GH_HOST_ACCOUNT for the GitHub account where
+# the source code repository is held, typically your username or the 
+# organization that you're part off.
+# MYPROJECT is the name of your project (the repository name and here
+# we assume that this is also the PyPi/conda package name) whereas 
+# MYPACKAGE is how you import it in python.
```
[the remark this answers is not in the document]
Comment (ricard1997) [reply mdakits/lipidorderkit/metadata.yaml]: Thanks for the feedback! I have added all the commits and remove explanatory comments, let me know if there is something else to be done.

**Comment by IAlibay** [comment]
CI is failing due to a typo - I opened a PR in the mdakit to fix this.

**Comment by ricard1997** [comment]
> CI is failing due to a typo - I opened a PR in the mdakit to fix this.

I have accepted the PR. Let me know if something else is needed.

File: mdakits/lipidorderkit/metadata.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (lilyminium) [reply mdakits/lipidorderkit/metadata.yaml:34]: I see, thanks for the update! Just to check -- @IAlibay is having future plans ok for the initial metadata file, or is it better to just say there's API docs here first and then open a new PR to update `metadata.yaml` once a user guide exists?

**Comment by lilyminium** [comment]
Thanks for updating the file @ricard1997! All looking pretty good -- however I think CI is failing due to not finding test files, which I hopefully fixed in https://github.com/ricard1997/lipidorderkit/pull/3.

**Comment by ricard1997** [comment]
> Thanks for updating the file @ricard1997! All looking pretty good -- however I think CI is failing due to not finding test files, which I hopefully fixed in [ricard1997/lipidorderkit#3](https://github.com/ricard1997/lipidorderkit/pull/3).

Thanks for the help with the problem with pytest!
I have solved the problem with the CI, and think everything is finally ready. Thanks for the feedback!

File: mdakits/lipidorderkit/metadata.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (lilyminium) [reply mdakits/lipidorderkit/metadata.yaml:34]: @IAlibay just pinging on this again

[the remark this answers is not in the document]
Comment (IAlibay) [reply mdakits/lipidorderkit/metadata.yaml:34]: Thanks for the ping @lilyminium - I saw the ping in my notifications but somehow not the comment

My recommendation would be to stick with API for now @ricard1997. I realise it's extra hoops, but it is best to be honest with users early on so you don't raise their expectations if they were to try to use/look at your code today.

That being said, if you really want to state it has a userguide, then the review process is not meant to be blocking so we won't halt over it.

[the remark this answers is not in the document]
Comment (ricard1997) [reply mdakits/lipidorderkit/metadata.yaml]: That makes sense. I have add a commit updating the documentation type. I'll soon add the user guide and do another commit on that. Thanks for the feedback!

**Review by lilyminium — APPROVED** [review]
All LGTM -- thank you for making the changes @ricard1997!! Very happy to see a new lipid kit in the registry.

**Pull request merged**